<ui version="4.0" >
 <class>WizardSubnetDNS</class>
 <widget class="QWidget" name="WizardSubnetDNS" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Vous pouvez specifier le domaine parent a utiliser par les ordinateurs clients sur le réseau pour la résolution de noms DNS.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Domaine parent :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineParentDomain" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Pour configurer les client d'etendue pour qu'ils utilisent les serveurs DNS sur le réseau, entrez les adresse IP pour ces serveurs.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Nom du serveur :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineDNSSolve" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>26</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="SpinnerWidget" native="1" name="spinnerWidget" >
           <property name="minimumSize" >
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnResolve" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Résoudre</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="AddressListEditor" native="1" name="dnsList" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AddressListEditor</class>
   <extends>QWidget</extends>
   <header>addresslisteditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpinnerWidget</class>
   <extends>QWidget</extends>
   <header>spinnerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
